server/application-server/src/main/java/de/tum/in/www1/hephaestus/chat/session/Session.java

```diff
@@ -0,0 +1,38 @@
+package de.tum.in.www1.hephaestus.chat.session;
+
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.lang.NonNull;
+import jakarta.persistence.*;
+import de.tum.in.www1.hephaestus.chat.message.Message;
+import de.tum.in.www1.hephaestus.gitprovider.user.User;
+
+@Entity
+@Table(name = "session")
+@Getter
+@Setter
+@ToString(callSuper = true)
+@NoArgsConstructor
+public class Session {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OrderColumn(name = "message_order")
+    @OneToMany(mappedBy = "session")
+    private List<Message> messages = new ArrayList<>();
+
+    @NonNull
+    @Column(name = "created_at")
```

Comment:
Just like in `Message`, you can omit `@Column` and make use of the automatic the naming.